libioc/Config/Jail/Globals.py

```diff
@@ -26,6 +26,7 @@
 
 
 DEFAULTS = libioc.Config.Data.Data({
+    "CONFIG_VERSION": 17,
```

Comment:
libioc does not read the `CONFIG_VERSION` at all, so should we not put it in the defaults, but allow this specific property to be ignored when found to be existing? Setting or changing it should not be possible and fail with the usual UnknownConfigProperty exception.

Reply:
`but allow this specific property to be ignored when found to be existing?`
I think unnecessary it.
Maybe, If it did implement, I think that code difficult to read.(If not used, that should be remove.)
Next commit, I will delete `CONFIG_VERSION`.
Regards.